samples/peripheral/802154_phy_test/src/ptt_conf.c

```diff
@@ -73,11 +74,15 @@ bool ptt_get_ant_mode_ext(uint8_t *ant_mode)
 
 #if CONFIG_PTT_ANTENNA_DIVERSITY
 #if CONFIG_PTT_ANT_MODE_AUTO
-	/* TODO: Implement when antenna diversity is supported in NCS */
+	nrf_802154_antenna_diversity_rx_mode_set(0x02);
+	nrf_802154_antenna_diversity_tx_mode_set(0x02);
 #elif CONFIG_PTT_ANT_MODE_MANUAL
-	/* TODO: Implement when antenna diversity is supported in NCS */
+	nrf_802154_antenna_diversity_rx_mode_set(0x02);
+	nrf_802154_antenna_diversity_tx_mode_set(0x02);
+
+	nrf_802154_antenna_diversity_rx_mode_set(0x01);
+	nrf_802154_antenna_diversity_tx_mode_set(0x01);
```

Comment:
Would it be possible to use some named values instead of magic numbers, as arguments of these functions?